src/SFA.DAS.FAA.Web/Views/Apply/TrainingCourses/List.cshtml

```diff
@@ -8,22 +8,23 @@
     <a class="govuk-back-link" href="@Url.RouteUrl(RouteNames.Apply, new { Model.ApplicationId })">Back</a>
 }
 
+<partial name="_ValidationSummary" />
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <partial name="_ValidationSummary" />
         <span class="govuk-caption-l">Education history</span>
         <h1 class="govuk-heading-l">Training courses</h1>
 
-        @if (Model.ShowTrainingCoursesAchieved)
+        @if (Model.ShowTrainingCoursesAchieved)
         {
-            @foreach (var trainingCourse in Model.TrainingCourses)
+            @foreach (var trainingCourse in Model.TrainingCourses)
             {
                 <div class="govuk-summary-card">
                     <div class="govuk-summary-card__title-wrapper">
                         <h2 class="govuk-summary-card__title">@trainingCourse.CourseName</h2>
                         <ul class="govuk-summary-card__actions">
                             <li class="govuk-summary-card__action">
-                                <a class="govuk-link" href="#">
+                                <a class="govuk-link" href="@Url.RouteUrl(RouteNames.ApplyApprenticeship.DeleteTrainingCourse, new {Model.ApplicationId, TrainingCourseId = trainingCourse.Id})">
```

Comment:
can we use `asp-route` and then `asp-route-applicationId=` and `asp-route-trianingCourseId=`